### PR TITLE
Additional RPC functions (and outputs) for more detail

### PR DESCRIPTION
**Pull request by galpHub**
### Usage

```bash
./divi-cli getexpandedstatistics
# Returns some staking metrics, 24h transaction metrics and block sizes, UTXO set counts as well as useless UXTO counts.
# Also total burned fees for blocks (excluding treasury and lottery blocks)

./divi-cli totalinputsforblock <blockHeight>
# Adds up the total of input amounts for the block at that height

./divi-cli gettxoutsetinfo
# Now includes the distribution of UTXOs binned by magnitude in increasing order from top to bottom. The topmost bin covers the range from 10^-8 to 10^-7 and each bin is one order of magnitude higher. Also includes a couple of metrics quantifying a portion of the bytes being spent for storing UTXOs (both useless and useful, i.e. value>0)
```

### CODE REVIEW

File: divi/src/txdb.cpp [hunk 1]
```diff
@@ -159,6 +161,19 @@ bool CCoinsViewDB::GetStats(CCoinsStats& stats) const
                         ss << VARINT(i + 1);
                         ss << out;
                         nTotalAmount += out.nValue;
+                        if(out.nValue>0)
```
Comment (galpHub) [inline divi/src/txdb.cpp]:
```suggestion
                        if(out.nValue > 0.0)
```

**Comment by 99darwin** [comment]
The commands are documented through the `help` function. We should plan to add the same information to wiki.diviproject.org but that is a separate issue for https://github.com/docs.

**Comment by ChrisCates** [comment]
@99darwin, yes, should get docs prepared for the wiki.

**Comment by ChrisCates** [comment]
Just added a new issue here: https://github.com/DiviProject/docs/issues/10

**Comment by galpHub** [comment]
> We need to document these RPC functions.
Done :D 🍡 
https://github.com/DiviProject/docs/pull/12

**Comment by ChrisCates** [comment]
@galpHub, everything looks good!
Just OCDing a bit about this.

```
****Note - for DIVI we added fCoinStake to the 2nd bit. Keep in mind when reading the following and adjust as needed.
```

Should just be

```
* Note - for DIVI we added fCoinStake to the 2nd bit. Keep in mind when reading the following and adjust as needed.
```

File: divi/src/coins.h [hunk 2]
```diff
@@ -18,7 +18,7 @@
 #include <boost/foreach.hpp>
 #include <boost/unordered_map.hpp>
 
-/** 
+/**
 
     ****Note - for DIVI we added fCoinStake to the 2nd bit. Keep in mind when reading the following and adjust as needed.
```
Comment (ChrisCates) [inline divi/src/coins.h]: Minor issue that just causes me some OCD!

**Pull request closed without merging**